apps/src/studio/studio.js

```diff
@@ -261,6 +261,23 @@ function loadLevel() {
   Studio.MAZE_HEIGHT = Studio.SQUARE_SIZE * Studio.ROWS;
   studioApp.MAZE_WIDTH = Studio.MAZE_WIDTH;
   studioApp.MAZE_HEIGHT = Studio.MAZE_HEIGHT;
+
+  if (skin.wallMapLayers) {
+    Studio.wallMapLayers = {};
+    for (let map in skin.wallMapLayers) {
```

Comment:
Nit: `const map` would be appropriate here since we never modify this value within the block.  Could also name it `mapName` or `mapKey` or something, since it's actually a string value used as an object key, not a 'map' (data structure or image).